collectors/0/hadoop_resourcemanager.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+# This file is part of tcollector.
+# Copyright (C) 2010  The tcollector Authors.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version.  This program is distributed in the hope that it
+# will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser
+# General Public License for more details.  You should have received a copy
+# of the GNU Lesser General Public License along with this program.  If not,
+# see <http://www.gnu.org/licenses/>.
+
+import sys
+from collectors.lib.hadoop import Hadoop
+
+REPLACEMENTS = {
+}
+
+def main(args):
+    rm = Hadoop(replacements=REPLACEMENTS, name='resourcemanager', delay=90, port=8088)
+    return rm.loop()
+
```

Comment:
looks like a repeated pattern:

REPLACEMENTS = {}
def main(args):
  rm = Hadoop(...)
  return rm.loop()

I think there is an opportunity to factorize